### PR TITLE
Feat/1381 notifications fiat

### LINKED ISSUE #1381: Show fiat rates in notifications
### Is your feature request related to a problem? Please describe

We have now the option to show fiat amounts when we display satoshi amounts.
So far we do not do this in the notifications. we should also show the same amount information.

### Additional context

we need to convert the amount here: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/events/notifications.js#L13-L23

Probably it would be helpful if we refactor the currency converter first to do the api call and caching in a background script action?

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

- show fiat in notifications
- refactor rate to be sats- instead of a btc-rate to work directly with sats everywhere

### Link this PR to an issue

Fixes #1381

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes

- with fiat
  image
- no fiat
  image

### How has this been tested?

- manually (see screenshots)
- unit-test

### CODE REVIEW

**Review by im-adithya — APPROVED** [review]

File: src/common/utils/currencyConvert.ts [hunk 1]
```diff
@@ -3,25 +3,18 @@
...
-
-const satoshisToBitcoin = (amountInSatoshis: number | string) => {
-  return Number(amountInSatoshis) / numSatsInBtc;
+const satsToFiat = async (amount: number | string, rate: number) => {
+  return Number(amount) * rate;
 };
 
 const satoshisToFiat = async ({
-  amountInSats,
+  amount,
```
Comment (bumi) [inline src/common/utils/currencyConvert.ts:11]: I kinda like names that describe what is in there.

**Review by bumi — APPROVED** [review]
utACK

did you test it with currencies where the values are different? (e.g. different amount of decimal points or values?

**Comment by escapedcat** [comment]
> did you test it with currencies where the values are different? (e.g. different amount of decimal points or values?

No, thanks!
Currently this would be cut off:
image


We could move Fiat to next line:
image

File: src/common/utils/currencyConvert.ts (hunk 1 above)
Comment (escapedcat) [reply src/common/utils/currencyConvert.ts:11]: I thought we do not need it anymore, because we got rid of the "btc value". But I can keep `amountInSats`

**Comment by im-adithya** [comment]
Maybe something like this:
`✅ Payment [to XYZ] Successful`
`Amount: 321 sats (25.73 NGN)`
`Fee: 0 sats`

**Comment by escapedcat** [comment]
> Maybe something like this: `✅ Payment [to XYZ] Successful` `Amount: 321 sats (25.73 NGN)` `Fee: 0 sats`

Good idea, let me try if simple line-breaks work in there

File: src/common/utils/currencyConvert.ts (hunk 1 above)
Comment (bumi) [reply src/common/utils/currencyConvert.ts:11]: I don't care, was just a nit...if one reads it in a year one might not know it. - even though the function is called satoshisInFiat.

**Pull request merged**